homeassistant/components/flux_led/const.py

```diff
@@ -0,0 +1,24 @@
+"""Constants of the FluxLed/MagicHome Integration."""
+
+DOMAIN = "flux_led"
+
+API = "flux_api"
+
+ATTR_IDENTIFIERS = "identifiers"
+ATTR_MANUFACTURER = "manufacturer"
+ATTR_MODEL = "model"
```

Comment:
Use the constants from `homeassistant/const.py` instead of adding integration constants for `ATTR_*`.